.travis.yml

```diff
@@ -0,0 +1,42 @@
+# references:
+# * http://www.objc.io/issue-6/travis-ci.html
+# * https://github.com/supermarin/xcpretty#usage
+
+language: objective-c
+
+# cache: cocoapods
+podfile: Example/Podfile
+
+before_install:
+  - brew install swiftlint
+  - gem install cocoapods --pre
+
+osx_image:
+  - xcode7.3
+  - xcode8
+# env:
+#   matrix:
+#     - TASK=build
+#     - TASK=lintpod
+
+matrix:
+    include:
+        - os: osx
+          osx_image: xcode8
+          env:
+            - TASK=build
+              DESTINATION="platform=iOS Simulator,name=iPhone 6,OS=10.0"
+        - os: osx
+          osx_image: xcode8
+          env: TASK=lintpod
+        # - os: osx
+        #   osx_image: xcode7.3
+        #   env:
+        #     - TASK=build
+        #       DESTINATION="OS=9.3,name=iPhone 6,platform=iOS Simulator"
+        - os: osx
+          osx_image: xcode7.3
```

Comment:
Xcode doesn't support Swift 2.3 nor 3.0, so there's no point to keeping this.
Did you consider to have CI for Swift 2.3 and 3.0?